plugins/CMakeLists.txt

```diff
@@ -41,6 +41,9 @@ add_subdirectory (exampleplugin)
 add_subdirectory (couchbase)
 add_subdirectory (sqs)
 add_subdirectory (spark)
+IF ( PLATFORM OR INCLUDE_EE_PLUGINS )
```

Comment:
It should only be checking INCLUDE_EE_PLUGINS.  Standard platform builds will not include these either.